posts/2023-09-29-open-liberty-on-aws-ecs-fargate.adoc

```diff
@@ -0,0 +1,684 @@
+---
+layout: post
+title: "Running Open Liberty on Amazon Elastic Container Service with AWS Fargate"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/abutch3r
+author_github: https://github.com/abutch3r
+seo-title: Running Open Liberty on Amazon Elastic Container Service with AWS Fargate - OpenLiberty.io
+seo-description: How to run Open Liberty in Amazon Elastic Container Service with AWS Fargate
+blog_description: "How to run Open Liberty in Amazon Elastic Container Service with AWS Fargate"
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Running Open Liberty on Amazon Elastic Container Service with AWS Fargate
+Alex Butcher <https://github.com/abutch3r>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+Serverless applications continue to be a hot topic in 2023. With MicroProfile being widely adopted by cloud native applications, many developers want to know:
+
+_How to run MicroProfile applications in Amazon Elastic Container Service in a Serverless manner?_
+
+This post demonstrates how to run a MicroProfile application on Open Liberty in Amazon Elastic Container Service with AWS Fargate.
+
+== What is Serverless?
+Serverless is short for serverless computing, which is an execution model in which the cloud provider allocates resources on demand. Serverless enables you to concentrate on your applications without needing to manage servers.
+
+It does not mean no server is running in the background. On the contrary, serverless architecture contains servers, however the cloud provider is responsible for provisioning, maintaining, and scaling the server infrastructure without operator interaction.
```

Comment:
Would combine this with the paragraph above and link with: "However, it does not mean...."